assistive-webdriver/rollup.config.js

```diff
@@ -16,7 +16,13 @@
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-import typescript from "@rollup/plugin-typescript";
+const instrumentation = process.env.ENABLE_COVERAGE === "1";
+if (instrumentation) {
+  require("./test/coverage/overrideReadFileSync");
+  console.log("WARNING: building with code coverage instrumentation!");
+}
+const typescript = require("@rollup/plugin-typescript");
+const plugins = () => [typescript({ module: "esnext", target: "es2017" })];
```

Comment:
what about just creating an army instead of a function ?
```js
const plugins = [typescript({ module: "esnext", target: "es2017" })]
```

and then use it like that:
```js
{
    plugins
}

Reply:
Thank you for this nice suggestion!
Actually, it does not only simplify the code, but it also and especially speeds up compilation, reducing the compilation time by 2.